### DISCUSSION TITLE
Is it mandatory to train teacher model for fastspeech?

### DISCUSSION THREAD

**Discussion by nellorebhanuteja** [espnet/espnet#5213] [General]
According to [docs](https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE/tts1#fastspeech-training), we need to train teacher model for training Fastspeech/Fastspeech2 systems.

Can I just get durations from MFA and do the training using the corresponding results?

**Comment by sw005320** [comment]
@kan-bayashi and @Fhrozen, can you answer it?

**Comment by Fhrozen** [comment]
> Is it mandatory to train teacher model for fastspeech?

I do not think so. I trained fastspeech2 and multiple Prodiff (NAR) using MFA alignments without any problem.
Just, I recommend training the MFA, until ESPnet pre-trained models are available.
